_issues/2019-11-22.md

```diff
@@ -0,0 +1,166 @@
+[Symmetry60](https://github.com/qmk/qmk_firmware/pull/7327) by marhalloweenvt<br>
+[sm68](https://github.com/qmk/qmk_firmware/pull/7279) by hineybush<br>
+[Retro75](https://github.com/qmk/qmk_firmware/pull/7294) by zvecr<br>
+[UA62](https://github.com/qmk/qmk_firmware/pull/7290) by Na-Cly<br>
+[Kudox Rev2](https://github.com/qmk/qmk_firmware/pull/7253) by x1-<br>
+[Kyria](https://github.com/qmk/qmk_firmware/pull/7222) by splitkb<br>
+[Hannah910](https://github.com/qmk/qmk_firmware/pull/7234) by mechlovin<br>
+[Slash](https://github.com/qmk/qmk_firmware/pull/7042) by asdftemp<br>
+[h88](https://github.com/qmk/qmk_firmware/pull/7201) by hineybush<br>
+[hbcp](https://github.com/qmk/qmk_firmware/pull/7101) by hineybush<br>
+[kbd67mkii soldered](https://github.com/qmk/qmk_firmware/pull/7202) by ai03-2725<br>
+[Savage65](https://github.com/qmk/qmk_firmware/pull/6968) by awkannan<br>
+[iron165](https://github.com/qmk/qmk_firmware/pull/6968) by awkannan<br>
+[tmo50 v2](https://github.com/qmk/qmk_firmware/pull/6968) by awkannan<br>
+[Think 6.5](https://github.com/qmk/qmk_firmware/pull/7199) by mechmerlin<br>
+[42](https://github.com/qmk/qmk_firmware/pull/7071) by nglgzz<br>
+[NEISO Macropad](https://github.com/qmk/qmk_firmware/pull/7163) by jrfhoutx<br>
+[M0116b](https://github.com/qmk/qmk_firmware/pull/7162) by jrfhoutx<br>
+[aranck](https://github.com/qmk/qmk_firmware/pull/7151) by Ardakilic<br>
+[Fruity60](https://github.com/qmk/qmk_firmware/pull/7021) by yanfali<br>
+[Naked64](https://github.com/qmk/qmk_firmware/pull/7144) by Salicyilc-acid3<br>
+[Sticc14](https://github.com/qmk/qmk_firmware/pull/7134) by ErkHal<br>
+[Shark PCB Rev Alpha](https://github.com/qmk/qmk_firmware/pull/7090) by Gondolindrim<br>
+[GTM Pad](https://github.com/qmk/qmk_firmware/pull/7123) by jrfhoutx<br>
+[hnah40rgb](https://github.com/qmk/qmk_firmware/pull/7083) by vuhokep<br>
+[Blackplum aka IMKG68](https://github.com/qmk/qmk_firmware/pull/7122) by eriqadams<br>
+[Clueboard California](https://github.com/qmk/qmk_firmware/pull/7127) by skullydazed<br>
+[uzu42](https://github.com/qmk/qmk_firmware/pull/6842) by nrtkbb<br>
+[Naked60](https://github.com/qmk/qmk_firmware/pull/6527) by Salicyilc-acid3<br>
+[BM43a](https://github.com/qmk/qmk_firmware/pull/6997) by mechmerlin<br>
+[Exent](https://github.com/qmk/qmk_firmware/pull/6985) by mechmerlin<br>
+[OSA](https://github.com/qmk/qmk_firmware/pull/6849) by jrfhoutx<br>
+[Phoebe](https://github.com/qmk/qmk_firmware/pull/6870) byMaxr1998<br>
+[Reviung34](https://github.com/qmk/qmk_firmware/pull/6847) by gtips<br>
+[Duck Orion V3](https://github.com/qmk/qmk_firmware/pull/6892) by mechmerlin/keebology<br>
+[Percent Skog Lite](https://github.com/qmk/qmk_firmware/pull/6882) by mechmerlin<br>
+[Stella](https://github.com/qmk/qmk_firmware/pull/6848) by vuhopkep<br>
+[Owlet60](https://github.com/qmk/qmk_firmware/pull/6803) by worthlessowl<br>
+[Wraith](https://github.com/qmk/qmk_firmware/pull/6810) by amberstarlight<br>
```

Comment:
`<br>` may work on GitHub but it doesn't work in Reddit Markdown. This section should be changed to a list IMO.

Reply:
Ooof I didn't know about reddit markdown =(